lib/src/main/scala/spinal/lib/bus/amba4/axi/sim/AxiMemorySim.scala

```diff
@@ -205,6 +205,13 @@ case class SparseMemory() {
     println(s"Loading 0x${byteArray.length.toHexString} bytes from ${file} to 0x${address.toHexString}")
   }
 
+  def loadHex(offset: Int, file: String) : Unit = {
```

Comment:
```suggestion
  def loadHex(file: String, offset: Int = 0) : Unit = {
```

Wouldn't this order be more conventional (eg to match `readHexFile`) and having `offset` defaulting to 0 be a good thing?